src/openrct2/config/Config.cpp

```diff
@@ -811,9 +815,8 @@ bool config_find_or_browse_install_directory()
                     return true;
                 }
 
-                std::string message = String::StdFormat(
-                    "Could not find %s" PATH_SEPARATOR "Data" PATH_SEPARATOR "g1.dat at this path", installPath.c_str());
-                uiContext->ShowMessageBox(message);
+                format_string(cantFindG1Dat, 1024, STR_COULD_NOT_FIND_AT_PATH, &g1DatPath);
```

Comment:
Change to the `std::string` variation of `format_string`, then you don't need the fixed length buffer.